src/v/kafka/server/tests/fetch_plan_bench.cc

```diff
@@ -51,7 +51,11 @@ struct fixture {
 
 struct fetch_plan_fixture : redpanda_thread_fixture {
     static constexpr size_t topic_name_length = 30;
+#ifdef NDEBUG
```

Comment:
should this use `#ifdef SEASTAR_DEFAULT_ALLOCATOR` instead?

Reply:
well with `not`

Reply:
In the Bazel build it's a local defines so it's only private to seastar .cc files: https://github.com/redpanda-data/redpanda/blob/5278835cc20ed853b80b318552e492effcaa778d/bazel/thirdparty/seastar.BUILD#L607

Reply:
hmm we do already use this in the batch cache to make reclaim work in debug.

Reply:
I also couldn't find a way to ask seastar if it was using it's own allocator or not at first glance

Reply:
Oh I see in the batch cache code we rely on that macro, I'll move it to be non-private